modules/olmv1-version-range-support.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.adoc
+
+:_mod-docs-content-type: CONCEPT
+
+[id="olmv1-version-range-support_{context}"]
+= Support for version ranges
+
+In {olmv1-first} you can specify a version range by using a comparison string in an Operator or extension's custom resource (CR). If you specify a version range in an Operator's CR, {olmv1} installs or updates the latest version of the Operator that can be resolved within the version range.
```

Comment:
Possible a reader might see "an Operator or extensions' custom resource (CR)" first, and then read "in the Operator's CR" later, and wonder "is an extension CR different?" Or maybe that "reader" is just "me". 😅 

```suggestion
In {olmv1-first} you can specify a version range by using a comparison string in an Operator or extension's custom resource (CR). If you specify a version range in the CR, {olmv1} installs or updates the latest version of the Operator that can be resolved within the version range.
```